es5-rest-persistence/README.md

```diff
@@ -0,0 +1,15 @@
+# ES5 REST persistence
+
+This module provides ES5 REST persistence instead of relying on the Transport protocol.
+
+## Usage
+
+To use this module, set the following values in your conductor configuration:
```

Comment:
Apart from below configuration, This module also has to be injected into Server module (Preferably replacing existing ES5 module, so to avoid dependency conflicts. Before Conductor V2.0, ES2 is shadowed to avoid this).

Reply:
So right now, I use a different package name for the ElasticSearchModuleV5Rest. It's under the `com.netflix.conductor.dao.es5rest`. This is because the es5rest depends on the `conductor-es5-persistence` for the `query.parser` logic as well as the `EmbeddedElasticSearchV5`. Because of the way gradle ties the dependencies of the various modules together (via the versionsOfDependencies.gradle file); there shouldn't be a dependency issue as they are relying on the same ES version.

That being said... maybe I should instead push the REST version of the IndexDAO inside the `conductor-es5-persistence` project? I don't have much experience with guice, but it seems like I can simply pass a REST flag to the `ElasticSearchModuleV5()` and then bind the IndexDAO to the rest version if the flag exists?

Reply:
Actually, that makes much better sense. There is no reason to maintain a separate module, if we can have ElasticSearchModuleV5Rest within module. The server module can still install one of them based on config.

Reply:
ok sounds good. This should be an easy change to make. Will get it up later today.

Reply:
Ok this is done the REST DAO is now inside the es5-persistence module now.